influunt-api/modules/influunt-core/app/models/Plano.java

```diff
@@ -334,7 +334,8 @@ public boolean isNumeroEstagiosEmModoManualOk() {
     @AssertTrue(groups = PlanosCheck.class,
         message = "Todos os grupos semafóricos devem possuir configurações de ativado/desativado.")
     public boolean isQuantidadeGrupoSemaforicoIgualQuantidadeAnel() {
-        return !(this.getGruposSemaforicosPlanos().isEmpty() || this.getAnel().getGruposSemaforicos().size() != this.getGruposSemaforicosPlanos().size());
+        List<GrupoSemaforicoPlano> grupoSemaforicoPlanos = this.getGruposSemaforicosPlanos().stream().filter(gp -> !gp.isDestroy()).collect(Collectors.toList());
+        return !(this.getGruposSemaforicosPlanos().isEmpty() || this.getAnel().getGruposSemaforicos().size() != grupoSemaforicoPlanos.size());
```

Comment:
Aqui no retorno ao invés de `this.getGruposSemoforicosPlanos().isEmpty()` acho que deveria ser `grupoSemoforicoPlanos.isEmpty()`. Se ele estiver com o `destroy = true` o grupo semafórico plano não deve ser considerado.